src/types/libdef/npm/jest_v22.x.x.js

```diff
@@ -66,14 +72,20 @@ type JestMockFn<TArguments: $ReadOnlyArray<*>, TReturn> = {
   /**
    * Sugar for only returning a value once inside your mock
    */
-  mockReturnValueOnce(value: TReturn): JestMockFn<TArguments, TReturn>
+  mockReturnValueOnce(value: TReturn): JestMockFn<TArguments, TReturn>,
+  mockResolvedValue(
+    value: ExtractPromiseValue<TReturn>
+  ): JestMockFn<TArguments, TReturn>,
+  mockResolvedValueOnce(
+    value: ExtractPromiseValue<TReturn>
+  ): JestMockFn<TArguments, TReturn>,
```

Comment:
This is the important stuff.
This is not working as expected really. I fired a PR at flow-typed: https://github.com/flowtype/flow-typed/pull/1918